ovos_solver_openai_persona/engines.py

```diff
@@ -146,6 +146,9 @@ def _do_streaming_api_request(self, messages):
             if chunk:
                 chunk = chunk.decode("utf-8")
                 chunk = json.loads(chunk.split("data: ", 1)[-1])
+                if chunk["error"] and chunk["error"]["message"]:
```

Comment:
should be `chunk.get("error")` or `"error" in chunk` as the key will be missing when things go as expected

Reply:
thanks, i am not a python pro. posting a revision which i'm hoping will be better; feel free to suggest more improvements